.env

```diff
@@ -7,3 +7,4 @@ MESSAGING_SENDER_ID=414830577349
 APP_ID=1:414830577349=web:eb834c0580bf7c3a
```

Comment:
kindly gitignore this file to avoid sharing senstive information

Reply:
Ohhh! I forgot to check if I ignore the .env file.
I'm updating that now.